docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "@KWSmit"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The CMUcam5 Pixy camera is available in a version designed for
+LEGO Mindstorms. In this tutorial we explain how to use it with ev3dev
+and python.
+
+General information about Pixy for LEGO Mindstorms can be found on
+Charmed Lab's [wiki-page](http://cmucam.org/projects/cmucam5/wiki/LEGO_Wiki).
+
+## What you need
+
+* LEGO Mindstorms EV3.
+* CMUcam5 Pixy for LEGO Mindstorms, connected to the EV3 with the cable
```

Comment:
I'm worried that people will not notice that you expect the cam to already plugged in. It seems obvious, but people can be forgetful :laughing: It might help to add a note about it somewhere else. Your choice though.